Sources/Core/Formatter/MultilineFormatter.swift

```diff
@@ -39,10 +39,31 @@ class MultilineFormatter: PrettyFormatter {
         if fields.count == 1, let field = fields.first {
             body = "\(field.0): \(field.1)"
         } else {
-            let contents = fields.map { "\($0): \($1)" }.joined(separator: ",\n")
-            body = contents.indentTail(size: prefix.count)
+            body = fields.map(indentInsertedKeyValueString(_:))
+                .joined(separator: ",\n")
+                .indentTail(size: prefix.count)
         }
 
         return prefix + body + ")"
     }
+
+    /// transform key value pair to single string inserted indent according to key
+    ///
+    /// ex:
+    /// ("owner", """
+    /// Owner(name: "Nanachi",
+    ///       age: 4)
+    /// """)
+    ///
+    /// goes to
+    ///
+    /// owner: Owner(name: "Nanachi",
+    ///              age: 4))
+    ///
+    /// - Parameter source: key value pair
+    private func indentInsertedKeyValueString(_ source: (String, String)) -> String {
```

Comment:
Specify arguments name to tuple to more readable ? if possible :)

e.g. `_ source: (label: String, value: String)`

Reply:
That is possible and applied👍🏻5e572e1be69eef60237f5417ecfeee671a552d66